seacrowd/sea_datasets/vintext/vintext.py

```diff
@@ -0,0 +1,214 @@
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_imtext",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema="seacrowd_imtext",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "image_path": datasets.Value("string"),
+                    "annotations": datasets.Sequence(
+                        {
+                            "x1": datasets.Value("int32"),
+                            "y1": datasets.Value("int32"),
+                            "x2": datasets.Value("int32"),
+                            "y2": datasets.Value("int32"),
+                            "x3": datasets.Value("int32"),
+                            "y3": datasets.Value("int32"),
+                            "x4": datasets.Value("int32"),
+                            "y4": datasets.Value("int32"),
+                            "transcript": datasets.Value("string"),
+                        }
+                    ),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_imtext":
+            features = schemas.image_text_features()
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        zip_filepath = os.path.join(os.path.dirname(__file__), "vietnamese_original.zip")
+        if not os.path.exists(zip_filepath):
+            gdown.download(id=_GDRIVE_ID, output=zip_filepath)
+
+        data_dir = dl_manager.extract(zip_filepath)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "imagepath": os.path.join(data_dir, "vietnamese/train_images"),
+                    "labelpath": os.path.join(data_dir, "vietnamese/labels"),
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "imagepath": os.path.join(data_dir, "vietnamese/test_image"),
+                    "labelpath": os.path.join(data_dir, "vietnamese/labels"),
+                },
+            ),
+        ]
+
+    def _generate_examples(self, imagepath: Path, labelpath: Path) -> Tuple[int, Dict]:
```

Comment:
I think `os.path.join` output type is a string, no? So this one should be changed even though there are no errors.